case_studies/lowell_high_school/03_second_meeting/planning.md

```diff
@@ -0,0 +1,7 @@
+For this meeting, we decided to do the Portfolio workshop. We also wanted to do
+a ice breaker game, and Jonathan suggested we try "The Big Wind Blows", so we 
+will try to play that.
+
+I anticipate that we are going to have problems getting people to start talking
```

Comment:
If you  forsee this being an issue, what will you do to combat this issue/ increase the conversation (beyond `The West Wind Blows`)?